src/tests/ftest/launch.py

```diff
@@ -886,6 +895,13 @@ def run_tests(test_files, tag_filter, args):
                                          args):
                     return_code |= 256
 
+    test_logs_lnk = os.path.join(avocado_logs_dir, "latest")
+    try:
+        os.remove(test_logs_lnk)
+        print("Removing sym link latest")
```

Comment:
Let's remove this.  Our test output is already chatty enough about things we don't really need to know.

Reply:
ok